pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/AggregationFunctionFactory.java

```diff
@@ -299,13 +299,21 @@ public static AggregationFunction getAggregationFunction(FunctionContext functio
             return new FourthMomentAggregationFunction(firstArgument, FourthMomentAggregationFunction.Type.KURTOSIS);
           case FOURTHMOMENT:
             return new FourthMomentAggregationFunction(firstArgument, FourthMomentAggregationFunction.Type.MOMENT);
+          case DISTINCTCOUNTTUPLESKETCH:
+            // mode actually doesn't matter here because we only care about keys, not values
+            return new DistinctCountIntegerTupleSketchAggregationFunction(arguments, IntegerSummary.Mode.Sum);
+          case DISTINCTCOUNTRAWINTEGERSUMTUPLESKETCH:
+            return new IntegerTupleSketchAggregationFunction(arguments, IntegerSummary.Mode.Sum);
+          case SUMVALUESINTEGERSUMTUPLESKETCH:
+            return new SumValuesIntegerTupleSketchAggregationFunction(arguments, IntegerSummary.Mode.Sum);
+          case AVGVALUEINTEGERSUMTUPLESKETCH:
+            return new AvgIntegerTupleSketchAggregationFunction(arguments, IntegerSummary.Mode.Sum);
```

Comment:
`DISTINCTCOUNTRAWINTEGERSUMTUPLESKETCH` is consistent with the current naming scheme but it's somewhat redundant to have the `DISTINCTCOUNT` prefix since it's clear that we are using a tuple (or theta) sketch.    This is just a subjective remark and not necessary to change it since this is consistent with the theta naming.

Reply:
yeah that is currently like that to line up with the other functions, maybe `UNION_SUM_TUPLE_SKETCH` would be cleaner

Reply:
I removed the `DistinctCount` prefixes from the other ones, so it's just `SUMVALUESINTEGERSUMTUPLESKETCH` and `AVGVALUEINTEGERSUMTUPLESKETCH` for the Sum and Average operations